examples/selection_tools/linear_region_scatter.py

```diff
@@ -0,0 +1,66 @@
+"""
+LinearRegionSelectors with ScatterGraphic
+=========================================
+
+Example showing how to use a `LinearRegionSelector` with a scatter plot. We demonstrate two use cases, a horizontal
+LinearRegionSelector which selects along the x-axis and a vertical selector which moves along the y-axis.
+"""
+
+# test_example = false
```

Comment:
make this `true`, we'd want to have a ground truth screenshot for this since it's a new feature.